src/lib/components/common/item_holder_metadata.ts

```diff
@@ -58,10 +65,17 @@ export abstract class ItemHolderMetadata extends FloatElement {
     protected render(): HTMLTemplateResult {
         if (!this.itemInfo) return html``;
 
+        const fadePercentage = this.asset && getFadePercentage(this.asset, this.itemInfo);
+
         return html`
             <span>
                 <span class="float">${formatFloatWithRank(this.itemInfo, 6)}</span>
-                <span class="seed">${formatSeed(this.itemInfo)}</span>
+                <span class="seed"
+                    >${formatSeed(this.itemInfo)}
+                    ${fadePercentage !== undefined
+                        ? html`<span class="fade">(${round(fadePercentage, 1)}%)</span>`
```

Comment:
This is nit-picky, but CS people care a lot about "100%" fades. Rounding 99.95% to 100% isn't considered a "true" fade and will probably lead to some people getting angry :)

Think we should floor to 1 decimal instead to 99.95% becomes 99.9%.

Reply:
got it, i wasn’t aware of this